Compass129Sources/COMfile/C12COM9.ASC

```diff
@@ -617,83 +617,83 @@ print_memory	push	hl
 print	LD	C,#09
 	JP	bdos
 
-;***************************************************memoryroutines
-;LET OP: er is een verschil tussen een tpa-segment en een tpaslot-segment
-;tpa-segment is dus (in veel gevallen) ramblok 0,1,2 of 3 van de primary mapper
-;tpaslot_segment kan elk ramblok van de primarymapper zijn
-
-src_mem	CALL	m_tabel	;maak tabel aan met segmenten
-	ret	c	;geen enkel tpaslotsegm gevonden
-	call	m_count_free	;als je nu geen 4 segmenten hebt, moet je
-	ld	hl,3	;er niet aan beginnen! De compassblocks
-	call	m_vgl_HLDE	;mogen immers niet in tpa-segmenten
-	jp	nc,m_freeall	;zitten, dus geen m_add_321 toepasbaar
+;*****************************************************memory routines
+;NOTE: there is a difference between a tpa segment and a tpaslot segment
+;tpa-segment is thus (in many cases) ramblock 0,1,2 or 3 of the primary mapper
+;tpaslot_segment can be any primarymapper ram block
+
+src_mem	CALL	m_tabel	;create table with segments
+	ret	c	;no tpaslotsegm found
+	call	m_count_free	;if you don't have 4 segments,
+	ld	hl,3	;don't start it! The compass blocks
+	call	m_vgl_HLDE	;after all, are not allowed in tpa segments
+	jp	nc,m_freeall	;sitting, so no m_add_321 applicable

```

Comment:
'sitting, ' is superfluous here. it's part of the 'mogen niet in ... zitten', which you already have suitably translated as 'are not allowed [to be] in ...'.